content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -219,11 +219,23 @@ v1.13.x, etcd v2 data MUST by migrated to the v3 storage backend, and
 kube-apiserver invocations changed to use `--storage-backend=etcd3`.
 
 The process for migrating from etcd2 to etcd3 is highly dependent on how the
-etcd cluster was deployed and configured, as well as how the Kubernetes 
+etcd cluster was deployed and configured, as well as how the Kubernetes
 cluster was deployed and configured. We recommend that you consult your cluster
 provider's documentation to see if there is a predefined solution.
 
 If your cluster was created via `kube-up.sh` and is still using etcd2 as its
 storage backend, please consult the [Kubernetes v1.12 etcd cluster upgrade docs](https://v1-12.docs.kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/#upgrading-and-rolling-back-etcd-clusters)
 
+## Known issues
+
+etcd servers may fail or disconnect briefly from kube-apiserver. Then, etcd client (embedded in kube-apiserver) should never deadlock being stuck with a downed node, and instead failover to other etcd nodes. etcd v3 client released in v3.3.13 (or before) had a critical bug that the client balancer failover does not properly work against secure endpoints. This affects kube-apiserver HA deployments.
+
+The fix was made in [etcd v3.4](https://github.com/etcd-io/etcd/pull/10911) (and backported to v3.3.14 or later): the new client now creates its own credential bundle to correctly set authority target in dial function.
+
+Because the fix requires gRPC dependency upgrade (to v1.23.0), downstream Kubernetes [did not backport etcd upgrades](https://github.com/kubernetes/kubernetes/issues/72102#issuecomment-526645978). Which means the [etcd fix in kube-apiserver](https://github.com/etcd-io/etcd/pull/10911/commits/db61ee106ca9363ba3f188ecf27d1a8843da33ab) is only available from Kubernetes 1.16.
+
```

Comment:
The current version of the k8s docs is 1.15, the last sentence, line 235, could be removed and added to the dev-1.16 branch and noted in the 1.16 release notes?

Should this issue be added to the 1.15 release notes or list of known v1.15 issues?

Suggested edit, something like:
To fix this bug for Kubernetes {{< latest-version >}} or earlier, build a custom `kube-apiserver`. You can make local changes to ... and follow these instructions ...

Reply:
> The current version of the k8s docs is 1.15, the last sentence, line 235, could be removed and added to the dev-1.16 branch and noted in the 1.16 release notes?

I am sure how we can do this cleanly when the PR is open to master branch.

We are including the fix in Kubernetes 1.16, so noting the version 1.15 specifically seems better?

Please let me know.

Reply:
@gyuho , I think your latest changes look good.